src/main/resources/config/application.yml

```diff
@@ -27,6 +27,10 @@ faf-api:
     base-url: ${NODEBB_BASE_URL:false}
     admin-user-id: ${NODEBB_ADMIN_USERID:1}
     master-token: ${NODEBB_MASTER_TOKEN:false}
+  gog:
+    token-format: ${GOG_TOKEN_FORMAT:FAF_%s}
+    profile-page-url: ${GOG_PROFILE_URL:https://www.gog.com/u/%s}
+    games-list-url: ${GOG_GAMES_LIST_URL:https://www.gog.com/u/%s/games/stats?sort=recent_playtime&order=desc&page=%d}
```

Comment:
you can remove sort and order params to make it more comprehensible (I tested it, it works)

Reply:
wouldn't it make finding FA more probable? or is it sorted that way by default?